components/select/select.tsx

```diff
@@ -10,12 +10,13 @@ import SelectDropdown from './select-dropdown'
 import SelectMultipleValue from './select-multiple-value'
 import Grid from '../grid'
 import { SelectContext, SelectConfig } from './select-context'
-import { getSizes } from './styles'
+import { getColors, getSizes } from './styles'
 import Ellipsis from '../shared/ellipsis'
 
 interface Props {
   disabled?: boolean
   size?: NormalSizes
+  status?: NormalTypes
```

Comment:
In the `Dot`, `Badge` and other components, Geist is now using `type` to specify the status, can we change the `status` to `type`? Consistent interface with other components.